src/common/SurgeStorage.cpp

```diff
@@ -354,13 +406,9 @@ void SurgeStorage::load_wt_wt(string filename, Wavetable* wt)
    wt_header wh;
    memset(&wh, 0, sizeof(wt_header));
 
-   // FIXME: Implement error handling when there is a convention to implement
-   // the error handling (e.g. return code or exception).
+   // fread(&wh, sizeof(wt_header), 1, f);
+   // if (wh.tag != vt_read_int32BE('vawt'))
```

Comment:
@baconpaul this line also failed to compile for me, unclear if it's necessary, but I didn't know what it was for

Reply:
I was just looking at that. It's checking the header is correct but we can just skip it because that comparison isn't quite right. Just make sure you don't bail out.

Reply:
```
diff --git a/src/common/SurgeStorage.cpp b/src/common/SurgeStorage.cpp
index ef8323c..0c3b826 100644
--- a/src/common/SurgeStorage.cpp
+++ b/src/common/SurgeStorage.cpp
@@ -406,10 +406,12 @@ void SurgeStorage::load_wt_wt(string filename, Wavetable* wt)
    wt_header wh;
    memset(&wh, 0, sizeof(wt_header));
 
-   // fread(&wh, sizeof(wt_header), 1, f);
+   size_t read = fread( &wh, sizeof( wt_header ), 1, f );
+   // I'm not sure why this ever worked but it is checking the 4 bytes against vawt so...
    // if (wh.tag != vt_read_int32BE('vawt'))
-   if (fread(&wh, sizeof(wt_header), 1, f) != 1)
+   if ( ! ( wh.tag[ 0 ] == 'v' && wh.tag[ 1 ] == 'a' && wh.tag[ 2 ] == 'w' && wh.tag[ 3 ] == 't' ) )
    {
+      // SOME sort of error reporting is appropriate
       fclose(f);
       return;
    }
```

I'm happy to apply this diff if you want. It recreates the intended behavior in a more likely to compile fashion. Let me know!

Reply:
ah yes, sounds good @baconpaul . feel free to do whatever you want to this PR, i'll be pretty busy the next few days

Reply:
OK cool I pushed that diff. I'm totally happy with the PR now, and will leave it unchanged from here. Hope we can get it merged! I'm off in branch land on vector graphics and MPE support, though, and that's a bit less important so won't do a PR any time soon. Have a great week and a happy holiday season.